.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,94 @@
+name: Pipeline deployment
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize]
+
+env:
+  ML_IMAGE_TAG: activelearning_ml:test
+  API_IMAGE_TAG: activelearning_api:test
+  ML_ENDPOINT: train_and_predict/
+  API: api/
+  WEBAPP: webapp/
+
+jobs:
+  linting:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8' 
+          architecture: 'x64'
+          cache: 'pip'
+      - name: flake8 Lint
+        uses: py-actions/flake8@v2
+        with:
+          ignore: "E402"
+          max-line-length: "120"
+          path: .
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
```

Comment:
I originally tried running npm install with node 16 on my local environment, and met with the following https://github.com/sass/node-sass/issues/2549. It appeared that for the node sass version specified in the package.json file would only be compatible with node 14, so I used that instead and managed to install the dependencies. I have updated the readme in the webapp section as well with the information, in case others might want to set it up locally as well.

Reply:
Got it. Another thing we can bump later.